test-framework/security/src/main/java/io/quarkus/test/security/QuarkusSecurityTestExtension.java

```diff
@@ -0,0 +1,61 @@
+package io.quarkus.test.security;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashSet;
+
+import javax.enterprise.inject.spi.CDI;
+
+import io.quarkus.security.runtime.QuarkusPrincipal;
+import io.quarkus.security.runtime.QuarkusSecurityIdentity;
+import io.quarkus.test.junit.callback.QuarkusTestAfterEachCallback;
+import io.quarkus.test.junit.callback.QuarkusTestBeforeEachCallback;
+import io.quarkus.test.junit.callback.QuarkusTestMethodContext;
+
+public class QuarkusSecurityTestExtension implements QuarkusTestBeforeEachCallback, QuarkusTestAfterEachCallback {
+
+    @Override
+    public void afterEach(QuarkusTestMethodContext context) {
+        CDI.current().select(TestAuthController.class).get().setEnabled(true);
```

Comment:
Shouldn't this be `false`?

Reply:
No, the default state is for security to be enabled.

Reply:
oops, I just realised that the default was false, so the first test would run with security disabled, I have fixed this.

Reply:
:+1: